tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -11,15 +11,13 @@ V := 1
 ARCH := $(shell uname -m)
 
 ifeq ($(ARCH), x86_64)
-EXTRA_TARBALL=cc-cloud-hypervisor-tarball \
+EXTRA_TARBALL=\
 	cc-tdx-kernel-tarball \
-	cc-tdx-qemu-tarball \
```

Comment:
ok, the commit message made me confused. We drop the CC target for tdx too but continue to build the same version from CCv0 (i.e. next merges main -> CCv0 we should keep the CCv0 version/source tree). And the other hypervisors we have the main and CCv0 version/source tree equal. Did I get it right?

Reply:
You got it right.

There are explicit comments in the versions.yaml file about the bits we cannot change.